### PR TITLE
[FIX] doc: minimum python version is 3.6

### PR DESCRIPTION
**Pull request by mart-e**
In 12.0 it was 3.5 but for 13.0, it is acceptable 3.6.
Debian Buster has 3.7 and Ubuntu 18.04 LTS has 3.6
This will avoid undeterminist errors with OrderedDict

CentOS 7 still does not ship python3 by default but it is possible to
install it
(e.g. https://www.digitalocean.com/community/tutorials/how-to-install-python-3-and-set-up-a-local-programming-environment-on-centos-7)
And CentOS 8 has been released 15 days ago [with python 3.6 in RHEL](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/8.0_release_notes/rhel-8_0_0_release#web-servers-databases-dynamic-languages)

Following discussion at https://github.com/OCA/maintainer-quality-tools/pull/613

cc @odony can we clean some [safe_eval 3.5 opcodes](https://github.com/odoo/odoo/blob/a5c4e262449fef03d05d5250e4523832db5fbf00/odoo/tools/safe_eval.py#L71) ?

### CODE REVIEW

File: odoo/__init__.py [hunk 1]
```diff
@@ -10,9 +10,9 @@
 #----------------------------------------------------------
 __path__ = __import__('pkgutil').extend_path(__path__, __name__)
 
-# As of version 12.0, python 2 is no longer supported, ensure py version is >= 3.5
+# As of version 12.0, python 2 is no longer supported, ensure py version is >= 3.6
```
Comment (fmdl) [inline odoo/__init__.py]:
```python
# As of version 12.0, python 2 is no longer supported, version 13.0 required Python >= 3.6
```

**Comment by d-fence** [comment]
@mart-e maybe change the commit message like "[FIX] doc,packaging: ..." to reflect the fact that it's not only a documentation change. I din't notice during the review.

File: odoo/__init__.py (hunk 1 above)
Comment (odony) [reply odoo/__init__.py]: just drop that comment ;)

**Comment by mart-e** [comment]
@robodoo r+

**Pull request closed without merging**